### PR TITLE
Observers and Umbrellas

### PR DESCRIPTION
**Pull request by nelsonic**
This code take you through Chapter 12 which does not have a "listing" on the Book's Website.
I've done all the steps in the book. and then filled in the gaps to make it pass all tests. 😉

### CODE REVIEW

**Comment by nelsonic** [comment]
`find and replace` all instances of `Rumbl.InfoSys` with `InfoSys`:

image

**Comment by nelsonic** [comment]
This is why Erlang is sick:
image
The "***observer***" is included for ***FREE***! 😍 
If you're still in "Node Land" there's: https://github.com/tableflip/guvnor which does an _excellent_ job of monitoring and re-starting node processes.

**Comment by nelsonic** [comment]
Followed the instructions on p.234 _carefully_ ... but now when I attempt to run `mix test` I get:
image

**Comment by nelsonic** [comment]
Totes completed the migration! 
image

File: rumbrella/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Rumbrella
+
+**TODO: Add description**
```
Comment (iteles) [inline rumbrella/README.md]: Could we just add a reference to the book's chapter for now?

File: rumbrella/apps/info_sys/config/config.exs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+use Mix.Config
+
+# This configuration is loaded before any dependency and is restricted
+# to this project. If another project depends on this project, this
+# file won't be loaded nor affect the parent project. For this reason,
+# if you want to provide default values for your application for
```
Comment (iteles) [inline rumbrella/apps/info_sys/config/config.exs]: :+1:

**Comment by shouston3** [comment]
Looks great, can't see any mistakes 👍  passing to @SimonLab to merge

**Comment by iteles** [comment]
Merging as we have 2 reviewer approvals - @danwhy @SimonLab you might want to take a look at this too :blush:

**Pull request merged**